specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1446,11 +1454,15 @@ components:
           - $ref: "#/components/schemas/Waardelijst"
           - description: "Mogelijke waarden zijn te vinden in deze [Waardelijst](http://www.kadaster.nl/schemas/waardelijsten/CultuurcodeBebouwd/)"
         aardCultuurOnbebouwd:
-          $ref: "#/components/schemas/Waardelijst"
+          allOf:
+          - $ref: "#/components/schemas/Waardelijst"
+          - description: "Mogelijke waarden zijn te vinden in deze [Waardelijst](http://www.kadaster.nl/schemas/waardelijsten/CultuurcodeOnbebouwd/)"
```

Comment:
Dit verschilt met de BAG. Daar hebben we ervoor gekozen dat we de allOf niet hiervoor te gebruiken. In OAS 3.1. is het wel toegestaan om een description als sibling aan een $ref toe te voegen

Reply:
Aangepast